modules/dxp/apps/analytics/analytics-reports-web/test/js/components/TrafficSources.js

```diff
@@ -37,7 +37,7 @@ describe('TrafficSources', () => {
 						],
 						helpMessage: 'Testing Help Message',
 						name: 'testing',
-						share: 30,
+						share: 30.0,
```

Comment:
As you know, in JS we don't have integers and floats, just numbers, so these are the same (`30 === 30.0` → `true`).

Reply:
Yeah, I know. I put the format that I received from the backend :)